aml-linking.md

```diff
@@ -0,0 +1,156 @@
+## Document nodes linking
+In AML it is possible to link nodes directly in the parsed graph. This is done by enabling linking in the dialect 
+definition. Example of linked nodes can be:
+* Nodes from the same document
+* Nodes from other documents parsed with the same dialect
+* Nodes form other documents parsed with a different dialect
+
+AML linking is enabled in the dialect definition using the `isLink: [boolean]` facet for a *property mapping*. 
+
+### Definitions
+#### Link target
+*Target node*: is the node which will be the target of the link property
+
+*Target document*: is the document which defines the target node
+
+*Target node mapping*: is the node mapping responsible for parsing the target node. It is the range of the link property
+mapping. It can be defined either within the same dialect, in a dialect library or in a fragment.
+
+*Target dialect/library*: is the dialect/library which defines the target node mapping. If the target node mapping 
+is defined in a fragment, then it is the dialect/library that includes that fragment.
+
+#### Link source
+*Source node*: is the node which is parsed with the link property mapping
+
+*Source document*: is the document which defines the source node
+
+*Source node mapping*: is the node which defined the link property mapping
+
+*Source dialect/library*: is the dialect/library which defines the source node mapping. If the source node mapping 
+is defined in a fragment, then it is the dialect/library that includes that fragment.
+
+### Example usage
```

Comment:
Should we include the JSON-LD output in this first example?
At least we should mention that the link will generate, not only the @id but also the @class of the node.